src/Core/src/Core/IApplication.cs

```diff
@@ -32,6 +32,12 @@ public interface IApplication : IElement
 		/// <param name="window">The window to close.</param>
 		void CloseWindow(IWindow window);
 
+		/// <summary>
+		/// Request that the application brings to the front the given window.
+		/// </summary>
+		/// <param name="window">The window to bring to the front.</param>
+		void ActivateWindow(IWindow window);
```

Comment:
Could this have a default interface implementation to avoid needing to update types that don't need/care about this?

Reply:
In general I prefer to not have breaking changes, especially when it is easy to avoid it. But I don't have a long history of understanding the perspective of this from Xamarin/MAUI, so I don't have a _strong_ opinion on this.

I'd lean towards having a default implementation if it is 'safe' to do so. For example if this method were truly required to be implemented in a meaningful way, then adding a nonsensical default just to avoid a compilation breaking change would be ill-advised. But if this is a more "optional" type of method, then I think a default implementation would make sense: say maybe most people don't care (it didn't exist until now anyway, how _could_ they care?), and the ones that do care can add a useful implementation for their needs.